test/SpokePool.Deposit.ts

```diff
@@ -7,6 +7,7 @@ import {
   depositRelayerFeePct as relayerFeePct,
   maxUint256,
 } from "./constants";
+import { ZERO_ADDRESS } from "@uma/common";
```

Comment:
Ethers exports this directly so we can avoid a dependency on @uma/common here.

```suggestion

const { AddressZero: ZERO_ADDRESS } = ethers.utils;
```